spec/support/factory_girl.rb

```diff
@@ -0,0 +1,31 @@
+RSpec.configure do |config|
+  config.include FactoryGirl::Syntax::Methods
+end
+
+# Test::Unit
+class Test::Unit::TestCase
+  include FactoryGirl::Syntax::Methods
+end
+
+# Cucumber
+World(FactoryGirl::Syntax::Methods)
+
+# Spinach
+class Spinach::FeatureSteps
+  include FactoryGirl::Syntax::Methods
+end
+
+# MiniTest
+class MiniTest::Unit::TestCase
```

Comment:
Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.